Habitica/src/com/habitrpg/android/habitica/ui/activities/TaskFormActivity.java

```diff
@@ -366,6 +425,15 @@ private void populate(Task task) {
         taskNotes.setText(task.notes);
         taskValue.setText(String.format("%.0f", task.value));
 
```

Comment:
Helps with visually loading in the Tags that were set by the user for a specific Task